pkg/driverbuilder/builder/ubuntu.go

```diff
@@ -75,10 +74,29 @@ func (v ubuntu) TemplateData(c Config, kr kernelrelease.KernelRelease, urls []st
 		KernelDownloadURLS:   urls,
 		KernelLocalVersion:   kr.FullExtraversion,
 		KernelHeadersPattern: headersPattern,
-		GCCVersion:           ubuntuGCCVersionFromKernelRelease(kr),
 	}
 }
 
+func (v ubuntu) GCCVersion(kr kernelrelease.KernelRelease) float64 {
```

Comment:
Ubuntu implements `GCCVersionRequestor` interface!

Reply:
Much cleaner to have this interface than e.g. `ubuntuGccVersionFromKernelRelease` =)